docs/community/faq.md

```diff
@@ -1,3 +1,71 @@
 # Frequently Asked Questions
 
-We collect frequently asked questions from issues/user feedbacks here.
+ - [General Questions](#general-questions)
+ - [Installation](#installation)
+ - [How to set up jobs](#how-to-set-up-jobs)
+ - [Failed jobs](#failed-jobs)
+ - [Miscellaneous](#miscellaneous)
+
+
+## General Questions
+
+**1. What are the merits of ABACUS in respect of functionality, performance, and/or accuracy?**
+
+Users are referred to the introduction of features of ABACUS in the [feature list](http://abacus.ustc.edu.cn/features/list.htm).
+
+## Installation
+
+
+## How to set up jobs
```

Comment:
How about changing this subject to "setting up jobs" conforming to other parts using a noun?

Reply:
OK, agree.